### PR TITLE
[inductor] Register an op for mm_plus_mm

### PR DESCRIPTION
**Pull request by desertfire**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #104835

Summary: Currently the aten version of mm_plus_mm has no cpp
implementation, and thus cpp_wrapper can not generate the correct cpp
function call for it.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8

Differential Revision: [D47372057](https://our.internmc.facebook.com/intern/diff/D47372057)

### CODE REVIEW

File: torch/csrc/inductor/inductor_ops.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using namespace at;
+
+Tensor _mm_plus_mm(
+    const Tensor& a,
+    const Tensor& b,
+    const Tensor& c,
+    const Tensor& d,
+    Tensor& out) {
+  at::matmul_out(out, a, b);
+  out.addmm_(c, d);
```
Comment (lezcano) [inline torch/csrc/inductor/inductor_ops.cpp:16]: Why are you using matmul for one and mm (or addmm) for the other one? We should use either mm for both or matmul for both. I think using `mm` for both would be best here.

**Review by jansel — APPROVED** [review]

**Review by SherlockNoMad — APPROVED** [review]

**Comment by desertfire** [comment]
@desertfire has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D47372057).

**Comment by desertfire** [comment]
@desertfire has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D47372057).

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: module 'os' has no attribute 'exit'

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: Object of type dict_keys is not JSON serializable

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: module 'os' has no attribute 'exit'

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: module 'os' has no attribute 'exit'

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: module 'os' has no attribute 'exit'

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Comment by huydhn** [comment]
## Reverting PR 104835 failed
**Reason**: module 'os' has no attribute 'exit'

**Comment by huydhn** [comment]
@pytorchbot successfully started a revert job. Check the current status [here](None).
Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)

**Pull request closed without merging**